backend/zf/settings.py

```diff
@@ -33,6 +33,9 @@
 ]
 
 CMS_TEMPLATES = [
+    ('base.html', 'Base page template'),
```

Comment:
The base template should probably not be available for editors to use directly.

Reply:
Alright, I though about that but was unsure whether the existence of that template needs to be specified anywhere so I added it to be safe. Will remove it!